### PR TITLE
Fixed markdown app topbar offset

### LINKED ISSUE #1229: [IE11-QA] Line wrapping in Markdown Editor preview
IE11

Steps to recreate:
1) Opened txt file for editing
2) Write a few words
3) Hit Enter
4) Enter a few words

Actual result: Preview differs from the actual result 
Expected result: Preview is the same as the actual result 

image

### PR DESCRIPTION
**Pull request by Julian1998**
## Description

Fixed the markdown editor app bar offset unit to fix the components visibility 

## Related Issue

- Fixes #1229 

## How Has This Been Tested?

- Created a new .txt file
- Opened with markdown editor app
- Checked if the markdown input textarea got rendered fully

## Screenshots (if appropriate):
Bildschirmfoto von »2019-06-16 20-24-18«

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: apps/markdown-editor/src/MarkdownEditorTopbar.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <oc-topbar variation="secondary" uk-sticky="offset: 60">
+  <oc-topbar variation="secondary" uk-sticky="offset: 60px">
```
Comment (LukasHirt) [inline apps/markdown-editor/src/MarkdownEditorTopbar.vue]: The units wasn't the problem - uikit use pixels in default in `uk-sticky`. There has been a change in the way we scroll the app - top-bar and app-bar doesn't scroll at all. Only the content of app is scrolling thanks to class uk-overflow-auto. The right fix then is to remove the stickyness completely from that component.

Comment (Julian1998) [reply apps/markdown-editor/src/MarkdownEditorTopbar.vue]: Thanks! Will have a look at it.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary><p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary><p>










</p></details>

**Pull request merged**